platform/android/src/native_map_view.cpp

```diff
@@ -439,20 +439,20 @@ void NativeMapView::rotateBy(jni::JNIEnv&, jni::jdouble sx, jni::jdouble sy, jni
 }
 
 void NativeMapView::setBearing(jni::JNIEnv&, jni::jdouble degrees, jni::jlong duration) {
-    map->setBearing(degrees, mbgl::AnimationOptions{mbgl::Milliseconds(duration)});
+    map->jumpTo(mbgl::CameraOptions().withAngle(degrees), mbgl::AnimationOptions{mbgl::Milliseconds(duration)});
```

Comment:
Shouldn't this be `easeTo`? Anyway, I don't get how this compiles since jumpTo is defined as `void jumpTo(const CameraOptions&);`

Reply:
Ah, it got fixed in a later commit.